src/Selenium2Driver.php

```diff
@@ -863,6 +863,21 @@ public function keyUp($xpath, $char, $modifier = null)
     }
 
     /**
+     * Simulates keyboard typing with Syn.type().
+     * See: https://github.com/bitovi/syn/blob/master/src/key.js#L133
+     *
+     * @param string     $xpath
+     * @param string     $text  example: "Firstname\tLast\tmail@example.org\r"
+     *
+     */
+    public function type($xpath, $text)
```

Comment:
This method can be useful, but needs to be added to:

1. driver test suite
2. driver interface
3. at least to 2 other drivers